static/js/entry/public.js

```diff
@@ -164,6 +166,8 @@ jQuery(document).ready(function ($) {
     description: description,
     emailBody: description + CURRENT_PAGE_URL
   });
+  $('.rrssb-buttons').find('.rrssb-twitter a').attr('href',
```

Comment:
Why not do:
```js
const tweetUrl = `https://twitter.com/intent/tweet?text=${twitterDescription}%20${CURRENT_PAGE_URL}`;
document.querySelector('.rrssb-buttons .rrssb-twitter a').href = tweetUrl;
```

Seems like we're trying to move away from jQuery where possible, and this is nicer ES6 syntax.